docs/tools/cicd/integration_test_failures.md

```diff
@@ -0,0 +1,51 @@
+# Integration Test Failures
+
+There are two (2) to receive notifications of integration test failures. 
+One is in [MilMove CircleCI dashboard](https://app.circleci.com/pipelines/github/transcom/mymove) 
+and the other is from the [GitHub pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) page.
+
+## CircleCI
```

Comment:
For this, I think it's better to document the link to CI rather than include actual screenshots. It's less to update when our CI/CD system changes.